releases/release-1.34/release-team.md

```diff
@@ -0,0 +1,15 @@
+| Emeritus Adviser  | Frederico Muñoz ([@fsmunoz](https://github.com/fsmunoz) / Slack: `@fsmunoz`)                                       | N/A                                                                                                                                                                                                                                                                                                                                               |
+| Enhancements      | Jenny Shu ([@jenshu](https://github.com/jenshu) / Slack: `@Jenny Shu`)                                             | TBD                                                                                                                                                                                                                                                                                                                                               |
+| Communications    | Agustina Barbetta ([@aibarbetta](https://github.com/aibarbetta) / Slack: `@aibarbetta`)                            | TBD                                                                                                                                                                                                                                                                                                                                               |
+| Release Signal    | Rajalakshmi Girish ( [@Rajalakshmi-Girish](https://github.com/Rajalakshmi-Girish) / Slack: `@Rajalakshmi Girish` ) | TBD                                                                                                                                                                                                                                                                                                                                               |
+| Docs              | Michelle Nguyen ([@michellengnx](https://github.com/michellengnx) / Slack: `@Michelle Nguyen`)                     | TBD                                                                                                                                                                                                                                                                                                                                               |
+| Branch Manager    | Matteo Bianchi ( [@mbianchidev](https://github.com/mbianchidev) / Slack: `@mbianchidev`)                           | Jim Angel ( [@jimangel](https://github.com/jimangel) / Slack: `@jimangel`)                                                                                                                                                                                                                                                                        |
```

Comment:
@mbianchidev As you asked, added you as the BM and @jimangel as the shadow. Although it feels weird with Jim as a shadow given how many times he has been the lead. (But I'll let rel-mgmt figure out the structure)